app/views/job_templates/index.html.erb

```diff
@@ -4,8 +4,7 @@
 
 <% title _("Job Templates") %>
 <% title_actions(documentation_button_rex('3.1JobTemplates'),
-                 display_link_if_authorized(_("New Job Template"),
-                                            hash_for_new_job_template_path)) %>
+                 new_link(_("New Job Template"), hash_for_new_job_template_path)) %>
```

Comment:
no need to pass the hash here, it get automatically calculated according to the controller